src/gui/static/src/app/components/pages/send-skycoin/send-form-advanced/send-form-advanced.component.ts

```diff
@@ -358,4 +423,43 @@ export class SendFormAdvancedComponent implements OnInit, OnDestroy {
 
     return hoursSelection;
   }
+
+  private updateAvailableBalance() {
```

Comment:
Can you extract those form values into variables and merge those two same conditions into one?